.github/workflows/gke.yml

```diff
@@ -19,6 +19,9 @@ on:
 env:
   GKE_PROJECT: ${{ secrets.GKE_PROJECT }}
   GKE_EMAIL: ${{ secrets.GKE_EMAIL }}
+  KUBEWEEKLY_GITHUB_TOKEN: ${{ secrets.KUBEWEEKLY_GITHUB_TOKEN }}
+  KUBEWEEKLY_TELGRAM_CHATID: ${{ secrets.KUBEWEEKLY_TELGRAM_CHATID }}
```

Comment:
typo? `KUBEWEEKLY_TELEGRAM_CHATID` instead of `KUBEWEEKLY_TELGRAM_CHATID `

same with the line below

Reply:
oh typo, wait a minute